woocommerce/woocommerce
# Fix new account notice not showing in Checkout block.

In a recent PR (#48985), the functionality in #48673 got regressed, this PR restores it.

![image](https://github.com/user-attachments/assets/8211891e-327a-4118-a5fd-7bfd52d85e57)




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:
0. Make sure that "Send password setup link (recommended)" option is enabled in the Accounts setting page.
1. As a guest, add something to your cart and checkout using the block based checkout.
2. Ensure you toggle the "create an account" box on.
3. After placing the order, assuming you're using the block based order confirmation, you'll see a notice at the top of the screen indicating the account was created. Leave this page open in a browser window.
4. Go find the account email your new account received and click the link to reset/set a password.
5. Set the password for the account and login.
6. Refresh the order confirmation page from earlier and confirm the notice is now hidden.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

Fixes a regression in a new PR that's not yet shipped.

</details>
